sympy/physics/quantum/tests/test_qapply.py

```diff
@@ -46,15 +50,16 @@ def test_basic():
 
 def test_extra():
     extra = z.dual*A*z
-    assert qapply(Jz*po*extra) == hbar*po*extra
-    assert qapply(Jx*z*extra) == (hbar*po/sqrt(2) + hbar*mo/sqrt(2))*extra
+    assert qapply(Jz*po*extra, op_join=False) == hbar*po*extra # op_join
+    assert qapply(Jx*z*extra, op_join=False) == \
+        hbar*po/sqrt(2)*extra + hbar*mo/sqrt(2)*extra #2 op_join, no .expand()
     assert qapply(
-        (Jplus + Jminus)*z/sqrt(2)*extra) == hbar*po*extra + hbar*mo*extra
-    assert qapply(Jz*(po + mo)*extra) == hbar*po*extra - hbar*mo*extra
-    assert qapply(Jz*po*extra + Jz*mo*extra) == hbar*po*extra - hbar*mo*extra
-    assert qapply(Jminus*Jminus*po*extra) == 2*hbar**2*mo*extra
-    assert qapply(Jplus**2*mo*extra) == 2*hbar**2*po*extra
-    assert qapply(Jplus**2*Jminus**2*po*extra) == 4*hbar**4*po*extra
+        (Jplus + Jminus)*z/sqrt(2)*extra, op_join=False) == hbar*po*extra + hbar*mo*extra #2 no .expand(), op_join
+    assert qapply(Jz*(po + mo)*extra, op_join=False) == hbar*po*extra - hbar*mo*extra # op_join
+    assert qapply(Jz*po*extra + Jz*mo*extra, op_join=False) == hbar*po*extra - hbar*mo*extra # op_join
+    assert qapply(Jminus*Jminus*po*extra, op_join=False) == 2*hbar**2*mo*extra # op_join
```

Comment:
Perhaps rather than changing the test we could test both with and without `op_join=False` to show the difference.

Reply:
I wanted to use the original test results to demonstrate that qapply reproduces them exactly. Sometimes the old results cannot be reproduced (e.g. all those that had expansion artefacts in them like sqrt(j**2 + +j + m**2 + m)). But if it was possible just by using an option I chose that way instead of changing the test outcome.